documentation/modules/configuring/con-config-mirrormaker2-connectors.adoc

```diff
@@ -242,4 +242,35 @@ Specifically, ensure that the following properties have the same value across al
 * `topic.filter.class`
 
 For example, the value for `replication.policy.class` must be the same for the source, checkpoint, and heartbeat connectors.
-Mismatched or missing settings cause issues with data replication or offset syncing, so it's essential to keep all relevant connectors configured with the same settings.
\ No newline at end of file
+Mismatched or missing settings cause issues with data replication or offset syncing, so it's essential to keep all relevant connectors configured with the same settings.
+
+== Listing the offsets of MirrorMaker 2 connectors
+
+To list the offset positions of the internal MirrorMaker 2 connectors, use the same configuration that's used to manage Kafka Connect connectors.
+
+In this example, the `sourceConnector` configuration is updated to return the connector offset position.
+The offset information is written to a specified config map.
+
+.Example configuration for connector producers and consumers
+[source,yaml,subs="+quotes,attributes"]
+----
+apiVersion: {KafkaMirrorMaker2ApiVersion}
+kind: KafkaMirrorMaker2
+metadata:
+  name: my-mirror-maker2
+spec:
+  version: {DefaultKafkaVersion}
+  # ...
+  mirrors:
+  - sourceCluster: "my-cluster-source"
+    targetCluster: "my-cluster-target"
+    sourceConnector:
+      listOffsets:
+        toConfigMap:
+          name: my-connector-offsets
+        # ...    
+----
```

Comment:
Is it worth mentioning that the field in the config map is the connector name with `->` replaced by `--`?